resources/lang/en/patrols/forest/hunting/any.json

```
@@ -1683,6 +2213,19 @@
                 "exp": 20,
                 "weight": 20,
                 "stat_skill": ["CLEVER,1"],
+            "relationships": [
+                {
+                    "cats_from": ["some_clan"],
+                    "cats_to": ["s_c"],
+                    "values": ["like", "respect"],
+                    "mutual": false,
+                    "amount": 5,
+                    "log":
+                    {
+                        "cats_from": "from_cat was pleased to see to_cat bring something soft back to camp for the nursery."
```

```suggestion
                        "cats_from": "from_cat was pleased to see to_cat bring something soft for the nursery back to camp."
```
I think this is a bit easier to follow